### ISSUE TITLE
Fix macOS sniffer tests

### ISSUE THREAD

**Issue by maksimdrachov** [OpenCyphal/pycyphal#282]
![image](https://user-images.githubusercontent.com/39975120/226160799-f524184d-a5ef-4e37-afde-2c5fa99ef6d7.png)

**Comment by silverv** [comment]
This also just happens on a Windows machine

**Comment by pavel-kirienko** [comment]
Yeah,  this is one of the flaky tests. We should lift the priority of the
test process to the maximum and see if it helps.

**Comment by maksimdrachov** [comment]
Btw, on macOS this happens consistently (on every test run).

Will test Pavels suggestion in the evening. How does one set this priority?

**Comment by pavel-kirienko** [comment]
Please see the last item here https://pycyphal--293.org.readthedocs.build/en/293/pages/faq.html
